### PR TITLE
ADD ARS Auvergne-Rhône-Alpes 27/03

### PR DESCRIPTION
**Pull request by abdoulsn**

### CODE REVIEW

File: agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+date: 2020-03-26
```
Comment (nsocheleau) [inline agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml]: Petit erreur de date? 
`date: 2020-03-27`

File: agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    deces: 9
+  - nom: Cantal
+    code: DEP-15
+    hospitalises: 7
+    gueris: 2
+    deces: 0
+  - nom: Drôme
+    code: DEP-26
+    hospitalises: 129
+    gueris: 46
```
Comment (nsocheleau) [inline agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml]: gueris: 39

File: agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    deces: 20
```
Comment (nsocheleau) [inline agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml]: deces: 21

File: agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    gueris: 57
+    deces: 4
+  - nom: Loire
+    code: DEP-42
+    hospitalises: 269
+    gueris: 46
+    deces: 20
+  - nom: Haute-Loire
+    code: DEP-43
+    hospitalises: 114
```
Comment (nsocheleau) [inline agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml]: hospitalises: 14

File: agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    gueris: 12
+    deces: 1
+  - nom: Puy-de-Dôme
+    code: DEP-63
+    hospitalises: 28
+    gueris: 14
+    deces: 0
+  - nom: Rhône
+    code: DEP-69
+    hospitalises: 6749
```
Comment (nsocheleau) [inline agences-regionales-sante/auvergne-rhone-alpes/2020-03-27.yaml]: hospitalises: 749

**Comment by abdoulsn** [comment]
Merci @nsocheleau

**Pull request merged**